packages/nodejs/src/cli/diagnose.ts

```diff
@@ -416,5 +416,9 @@ export class Diagnose {
 }
 
 function format_value(value: any) {
-  return util.inspect(value)
+  if (typeof value == "object") {
```

Comment:
```suggestion
  if (typeof value === "object") {
```

Reply:
Thanks to the TS compiler we don't need to write equality operators as in vanilla JS

Reply:
It's not good practice to keep doing this? Does TS give you a warning if it could ever be ambiguous?